gui-tests/theme.goml

```diff
@@ -0,0 +1,18 @@
+// setting local-storage emulates how we detect rustdoc changing the theme, we
+// only run this detection on rustdoc pages so must change there before visiting
+// other pages
+
+// on rustdoc pages we only control the .nav-container and its descendants, on
+// the crate page we control the whole page
+
+go-to: |DOC_PATH| + "/sysinfo"
+set-local-storage: { "rustdoc-theme": null }
+assert-css: (".nav-container", { "background-color": "rgb(255, 255, 255)" })
```

Comment:
If there is an event triggered when the local storage is updated, might be better to use `wait-for-css` instead (only the command needs to be updated) in case there is a delay before the color is changed.